packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "04085c80c2c84ea6b8375042d61797f7f46138fd",
-    "ref_origin" : "dcos-mesos-master-f179400b"
+    "ref": "9d1cb5ecba1a7921b067ad94092c3663adf19b24",
+    "ref_origin" : "dcos-mesos-1.2.0-rc1"
```

Comment:
Ah, OK, I thought it was supposed to reference the SHA in `mesosphere/mesos` that matched the equivalent SHA in upstream`mesos` on top of which the DC/OS specific commits have been cherry-picked. You're saying it should just be the HEAD of the branch that includes the cherry-picked commits?